File: testdata/repos/commons/templates/create_formula/languages/node/helloWorld/src/formula/formula.js
const clc = require("cli-color")

function Run(inputText, inputBoolean, inputList, inputPassword) {
    console.log("Hello World!")
    console.log(clc.green("My name is "+ inputText +"."))
    if(inputBoolean){
        console.log(clc.blue("I've already created formulas using Ritchie."))
    } else {
        console.log(clc.red("I'm excited in creating new formulas using Ritchie."))
    }
    console.log(clc.yellow("Today, I want to automate "+ inputList +"."))
    console.log(clc.cyan("My secret is " + inputPassword +"."))
}

const formula = Run
module.exports = formula

File: testdata/repos/commons/templates/create_formula/languages/python2/helloWorld/src/formula/formula.py
#!/usr/bin/python2
from colored import fg, attr
from distutils.util import strtobool


def Run(input1, input2, input3, input4):
    print "Hello World!"
    print "%sMy name is %s.%s" % (fg(2), input1, attr(0))

    if strtobool(input2):
        print "%sI've already created formulas using Ritchie.%s" % (fg(3), attr(0))
    else:
        print "%sI'm excited in creating new formulas using Ritchie.%s" % (
            fg(3),
            attr(0),
        )
    print "%sToday, I want to automate %s.%s" % (fg(1), input3, attr(0))
    print "%sMy secret is '%s'.%s" % (fg(3), input4, attr(0))
